src/Pelagos/Bundle/AppBundle/Controller/UI/PersonController.php

```diff
@@ -56,8 +58,13 @@ public function defaultAction($id = null)
                     $ui['PersonResearchGroupForms'][$personResearchGroup->getId()] = $formView;
                     $ui['PersonResearchGroupEditLabel'][$personResearchGroup->getId()]
                         = new EntityProperty($personResearchGroup, 'label');
+                    $listOfPersonsInResearchGroup = $personResearchGroup->getResearchGroup()->getPersonResearchGroups();
+                    foreach ($listOfPersonsInResearchGroup as $personResearch) {
```

Comment:
What is there are more? The last one will get picked, is this correct?

Reply:
Changed logic to show multiple